docs/api/buckets.rst

```diff
@@ -3,8 +3,7 @@
 Buckets
 #######
 
-A bucket is the parent object of collections and groups. It can be viewed as
-a namespace where all collections and groups are stored.
+A bucket is the parent object of collections and groups.
```

Comment:
Ah, we're on the API page, didn't notice! You're correct then :-) We should link to the concept then to make it easier to follow, probably.